packages/store-ui/src/atoms/List/List.tsx

```diff
@@ -0,0 +1,67 @@
+import type { ElementType, ComponentPropsWithRef, ReactElement } from 'react'
+import React, { forwardRef } from 'react'
+
+import type { PolymorphicComponentPropsWithRef } from '../../typings'
+
+export type ListVariant = 'description' | 'ordered' | 'unordered'
+
+const variantToType = (variant: ListVariant) => {
+  switch (variant) {
+    case 'description':
+      return 'dl'
+
+    case 'unordered':
+      return 'ul'
+
+    case 'ordered':
+      return 'ol'
+
+    default:
+      return 'ul'
+  }
+}
+
+interface Props {
+  testId?: string
+  variant?: ListVariant
+}
+
+export type ListProps<C extends ElementType> = PolymorphicComponentPropsWithRef<
+  C,
+  Props
+>
+
+type ListComponent = <C extends ElementType = 'ul'>(
+  props: ListProps<C>
+) => ReactElement | null
+
+const List: ListComponent = forwardRef(function List<
+  C extends ElementType = 'ul'
+>(
+  {
+    testId = 'store-list',
+    variant = 'unordered',
+    as,
+    ...rawProps
+  }: ListProps<C>,
+  ref: ComponentPropsWithRef<C>['ref']
+) {
+  const dataAttributes = {
+    'data-test-id': testId,
```

Comment:
This should be here?

Reply:
Yep, this is also a data attribute

Reply:
Oooh! Now I see hahahah I'll remove it from line 60